### ISSUE TITLE
Can't convert TeX math: \pmod

### ISSUE THREAD

**Issue by ErwinJanssen** [jgm/texmath#131]
Pandoc version:

```
pandoc 2.5
Compiled with pandoc-types 1.17.5.4, texmath 0.11.1.2, skylighting 0.7.5
```

The LaTeX math command `\pmod` (parenthesized modulo) results in an error. A minimal example:

```
~> echo '$a \equiv r \pmod{5}$' | pandoc
[WARNING] Could not convert TeX math 'a \equiv r \pmod{5}', rendering as TeX:
  a \equiv r \pmod{5}
                  ^
  unexpected "{"
  expecting "%", "\\label", "\\nonumber" or whitespace
<p><span class="math inline">$a \equiv r \pmod{5}$</span></p>
```

Removing the braces does not resolve the problem:

```
~> echo '$a \equiv r \pmod 5$' | pandoc
[WARNING] Could not convert TeX math 'a \equiv r \pmod 5', rendering as TeX:
  a \equiv r \pmod 5
                   ^
  unexpected "5"
  expecting "%", "\\label", "\\nonumber" or whitespace
<p><span class="math inline">$a \equiv r \pmod 5$</span></p>
```

The related command `\bmod` however, does work (both with and without braces).

**Comment by jgm** [comment]
I've added this to the todo list for the texmath library.
Note that, while you're waiting for texmath to be improved, you should be able to handle this by adding the following to your document before processing with pandoc:
```
\renewcommand{\pmod}[1]{(\mod #1)}
```
(untested)

**Comment by ErwinJanssen** [comment]
Thank you for your swift reply. The workaround works and provides acceptable output for now. I will look forward to the new release.

**Comment by Yuki-Nagato** [comment]
I am sorry but it is still not fixed.

**Comment by jgm** [comment]
Closed with commit 432c4fa06f3ae80c45a2f3a17e0b425e2dc01086
